modules/installation-dns-user-infra.adoc

```diff
@@ -12,6 +12,11 @@
 [id="installation-dns-user-infra_{context}"]
 = User-provisioned DNS requirements
 
+DNS is used for name resolution (A/AAAA or CNAME records) and reverse name resolution (PTR records).
```

Comment:
Generally because of the ISG, we try to avoid parenthetical statements. Would you mind rewriting those without parentheses?